docs/guides/code_push_release.md

```diff
@@ -0,0 +1,252 @@
+
+🚀 Ready to create a new release!
+
+📱 App: time_shift (51751336-6a7c-4972-b4ec-8fc1591fb2b3)
+📦 Release Version: 1.0.3+6
+🕹️ Platform: android (arm64, arm32, x86_64)
+
+Would you like to continue? (y/N) Yes
+✓ Fetching releases (70ms)
+✓ Fetching Flutter revision (22ms)
+✓ Creating release (61ms)
+✓ Creating artifacts (2.5s)
+
+✅ Published Release!
+
+Your next step is to upload the app bundle to the Play Store.
+./build/app/outputs/bundle/release/app-release.aab
+
+See the following link for more information:
+https://support.google.com/googleplay/android-developer/answer/9859152?hl=en
+```
+
+### Upload to the Play Store
+
+As per the instructions above, we must upload the generated `.aab` to the Play Store.
+
+1. Navigate to the [Play Console](https://play.google.com/console/developers).
+1. Choose your developer account (for us, it's Shorebirdbird.dev).
+1. Select the Time Shift app.
+1. Select "Testing -> Open Testing" from the side bar.
+1. Click the "Create new release" button.
+
+![CreateNewRelease](https://github.com/shorebirdtech/docs/assets/581764/90c9c7ed-bc39-4731-bfec-524f89e2baf6)
+
+Upload the `.aab` file located at `./build/app/outputs/bundle/release/app-release.aab`.
+
+1. From the root directory of the project, run `open ./build/app/outputs/bundle/release/` to open the folder containing the `.aab` in Finder.
+1. Drag `app-release.aab` into the Play Console to upload.
+
+Once the upload completes, the Play Store will correctly recognize the new version as `6 (1.0.3)`.
+
+![UploadedBundle](https://github.com/shorebirdtech/docs/assets/581764/1994cb5a-4cd6-4f1b-a88c-f5aaa3d1433d)
+
+Click "Next" and then "Save" (both in the bottom-right corner) to submit.
+
+This will take you to the publishing overview page. Click "Submit for review" to submit the release for review.
+
+You should now see a release in the Play Store console with an "In review" status:
+
+![InReview](https://github.com/shorebirdtech/docs/assets/581764/4cfdc7fb-2049-4110-b1cd-da99c7a491f7)
+
+Now you must wait for the Play Store to approve the release.
+
+### Create a GitHub release
+
+It's recommended to create a GitHub release as well.
+
+1. Navigate to https://github.com/shorebirdtech/time_shift/releases.
+1. Click "Draft a new release".
+1. Choose the tag we created earlier (`v1.0.3+6`).
+1. Title the release "v1.0.3+6".
+1. Add a description of the release ("Changes the default clock face to 'generative'").
+1. Publish the release.
+
+## After the release is approved
+
+Once the release has been approved, you will be able to download it from the Play Store.
+
+## Creating a patch
+
+Patches can be pushed to fix bugs in the `1.0.3+6` release without requiring a new submission to the Play Store.
+
+### Make the change
+
+Start by checking out the `v1.0.3+6` release tag:
+
+```sh
+git checkout v1.0.3+6
+```
+
+Next, create a branch, as this change will represent a divergence from the `main` branch:
+
+```sh
+git checkout -b v1.0.3+6-patch1
```

Comment:
with this approach we'll just end up with lots of branches right? Couldn't we also delete the branch after the patch has been tagged?

Reply:
Yeah, I could see this getting a bit messy. Added cleanup instructions.